packages/mux-uploader/test/web-test-runner.config.mjs

```diff
@@ -2,6 +2,22 @@ import { esbuildPlugin } from '@web/dev-server-esbuild';
 import { importMapsPlugin } from '@web/dev-server-import-maps';
 
 export default {
+  testFramework: {
+    config: {
+      ui: 'bdd',
+      timeout: '5000',
+    },
+  },
+  middleware: [
```

Comment:
thought(non-blocking): Fine for now, though this may be hard to follow/maintain longer term.

Reply:
I explored using a Sinon fake server and it just didn't seem possible since Upchunk's XHR was modifying and caching a version of `XMLHttpRequest` before Sinon could shim it

Reply:
it's possible to do so by overriding the global XHR before anything else is loaded.

Reply:
All good/definitely not a blocker. We may be able to apply similar acrobatics tricks that I ended up having to apply for `xhr-mock` in Upchunk (See, e.g.: https://github.com/muxinc/upchunk/blob/master/test/upchunk.spec.ts#L8-L19). I should be able to help investigate that to see if it's possible (either with sinon or simply again using `xhr-mock` for testing here, or if we want to just continue relying on the dev server middleware). Even if we stick with the middleware solution, we may want to break that stuff out into its own module? Either way, I don't think we should worry about it too much just yet.

Reply:
yeah, I don't think it's a blocker for this PR, but I do think longer term it would be good to set it up. Should be similar to who you have there with xhr-mock.

Reply:
I do think the sinon direction would be best because it makes the test self-contained as the "test server" is defined inside the test.